packaging/chocolatey/crc/crc.nuspec.in

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>crc</id>
+    <version>__VERSION__</version>
+    <owners>https://github.com/crc-org</owners>
+    <title>CRC - Runs Containers</title>
+    <authors>anjannath</authors>
+    <projectUrl>https://crc.dev/blog/about</projectUrl>
+    <iconUrl>https://i.ibb.co/m0b6dsL/crc-upstream-logo.png</iconUrl>

```

Comment:
Forgot to ask before, but can we add this to some git repo, or to crc.dev, in short to some place we have control over?

Reply:
this is supposed to be a CDN link to not rely on a single location, the current one is a random image hosting service i found online, but we can use more reliable free CDNs, the choco docs mentions some services but missed this in the PR

relevant choco docs link: https://docs.chocolatey.org/en-us/create/create-packages#package-icon-guidelines